### PR TITLE
Sandeep exeption handling changes

### PR DESCRIPTION
**Pull request by AppineersApps**
Sandeep exeption handling changes

### CODE REVIEW

File: application/webservice/basic_appineers_master/models/Users_model.php [hunk 1]
```diff
@@ -85,38 +107,32 @@ public function logout($params_arr = array(), $where_arr = array())
...
                 throw new Exception("Failure in updation.");
             }
             $result_param = "affected_rows";
             $result_arr[0][$result_param] = $affected_rows;
             $success = 1;
-        }
-        catch(Exception $e)
-        {
+        } catch (Exception $e) {
             $success = 0;
```
Comment (shrikantm09) [inline application/webservice/basic_appineers_master/models/Users_model.php:136]: exception call function missing here

File: application/webservice/basic_appineers_master/models/Users_model.php [hunk 2]
```diff
@@ -179,38 +189,32 @@ public function update_notification($params_arr = array(), $where_arr = array())
...
```
Comment (shrikantm09) [inline application/webservice/basic_appineers_master/models/Users_model.php:218]: exception missing

File: application/webservice/basic_appineers_master/controllers/Logout.php [hunk 3]
```diff
@@ -72,70 +82,60 @@ public function rules_logout($request_arr = array())
...
+            /*********unset user acces token and logout from application ***********/
             $input_params = $this->logout($input_params);
 
             $output_response = $this->users_finish_success($input_params);
             return $output_response;
-        }
-        catch(Exception $e)
-        {
+        } catch (Exception $e) {
             $message = $e->getMessage();
```
Comment (shrikantm09) [inline application/webservice/basic_appineers_master/controllers/Logout.php:112]: throw & add exception here.

File: application/webservice/basic_appineers_master/controllers/Post_a_feedback.php [hunk 4]
```diff
@@ -101,111 +111,99 @@ public function rules_post_a_feedback($request_arr = array())
...
-            {
+            if (isset($input_params["app_version"])) {
                 $params_arr["app_version"] = $input_params["app_version"];
             }
             $this->block_result = $this->user_query_model->post_a_feedback($params_arr);
-        }
-        catch(Exception $e)
-        {
+        } catch (Exception $e) {
             $success = 0;
```
Comment (shrikantm09) [inline application/webservice/basic_appineers_master/controllers/Post_a_feedback.php:207]: add exception here

**Review by shrikantm09 — COMMENTED** [review]
please resolve commented points

**Comment by AppineersApps** [comment]
update above changes plz check and merge

**Pull request merged**